Client/Frontend/Browser/SearchLoader.swift

```diff
@@ -32,7 +32,14 @@ class _SearchLoader<UnusedA, UnusedB>: Loader<Cursor<Site>, SearchViewController
 
     fileprivate lazy var topDomains: [String] = {
         let filePath = Bundle.main.path(forResource: "topdomains", ofType: "txt")
-        return try! String(contentsOfFile: filePath!).components(separatedBy: "\n")
+        let result: [String]
+        do {
+            result = try String(contentsOfFile: filePath!).components(separatedBy: "\n")
+        } catch {
+            Sentry.shared.send(message: "SearchLoader topDomains failed", tag: .general, severity: .error)
+            result = []
```

Comment:
Might be simpler to just `return []` here, and `return try String(…)` above, if Swift will let you do that.